csv-bulk-signature/src/main.ts

```diff
@@ -173,3 +175,24 @@ function checkHeadersToBeSame() {
       counter = 1;
     });
 }
+
+async function zipUpFile() {
+  const folderPath = "./dist";
+  const files = fs.readdirSync(folderPath);
+  const filesHtm = files.filter((file) => file.endsWith(".htm"));
+
+  const zip = new jszip();
+
+  filesHtm.forEach((file) => {
+    try {
+      const filePath = `${folderPath}/${file}`;
+      const fileContents = fs.readFileSync(filePath);
+      zip.file(file, fileContents);
+    } catch (error) {
+      console.error(`Error adding file ${file}: ${error.message}`);
+    }
+  });
+
+  const buffer = await zip.generateAsync({ type: "nodebuffer" });
+  fs.writeFileSync("signature-templates.zip", buffer);
```

Comment:
To keep track of signature builds, what if we remove `templates` and replace with a datetime?
`signatures-22-12-27-11:30.zip`

Reply:
👍